packages/mobile/src/navigator/NavigationService.ts

```diff
@@ -19,14 +19,15 @@ const TAG = 'NavigationService'
 type SafeNavigate = typeof navigate
 
 export const navigationRef = createRef<NavigationContainerRef>()
+export const isReadyRef: MutableRefObject<boolean | null> = createRef()
```

Comment:
Can we name this something a little more descriptive? Maybe `navigatorIsReadyRef`?

Reply:
+1